Frontend/app/shared/participant.ts

```diff
@@ -61,4 +61,20 @@ export class Participant{
         }
         return maxWest;
     }
+
+    getNumStories() : string {
```

Comment:
Also add conditions so that it displays 'story' instead of 'stories' if there is only 1.